### ISSUE TITLE
Make all grad_norm logs in a single section

### ISSUE THREAD

**Issue by V3RGANz** [PyTorchLightning/pytorch-lightning#3021]
## 🚀 Feature
Make all grad_norm logs in a single section

### Motivation

current tensorboard logs of gradient norm looks like:
<img width="512" alt="image" src="https://user-images.githubusercontent.com/14924440/90439605-03e7e480-e100-11ea-964f-80917deac708.png">
e.g. for each parameter, there is a separate section that contains a single plot. It takes too much space especially in case of really deep networks or when you're experimenting with different architectures. In both cases, it may take several "screens" and scrolling it becomes annoying. Even if you logged gradnorm in a single experiment and turn it off, this empty plots still exist.

### Pitch

instead of logging metrics with tag ``grad_norm_parameter_name`` make single section `grad_norm` by changing tag on `grad_norm/parameter_name`

or, change https://github.com/PyTorchLightning/pytorch-lightning/blob/ca18e11f6efe822098c79e7d9124b08a55bcd908/pytorch_lightning/trainer/logging.py#L60
to 
```python
metrics.update({f'grad_norm/{k}': v for k, v in grad_norm_dic.items()})
```
